src/main/scala/cromwell/binding/IOInterface.scala

```diff
@@ -0,0 +1,16 @@
+package cromwell.binding
```

Comment:
I think we need to start looking at the binding package as "if we were to construct a workflow out of scala objects from an arbitrary workflow language and for an arbitrary backend, what would we have in there?" - it's not there now but looking ahead it'll make life less painful to try to apply that reasoning going forward.